### DISCUSSION TITLE
some thoughts about 3d rendered wallpapers

### DISCUSSION THREAD

**Discussion by brigazvi** [elementary/wallpapers#175] [Ideas]
I have a little weird vision for elementary wallpapers that I cooking in my head for a long time.
don't know if its synchronize with the vision of the core developers, so I just dropping it here.

making wallpapers in 3d software like blender.

why?
in short, a lot of control.

1. a lot of control over the composition. making sure that the top of the image is clean and have the same brightness all over the horizontal line, so the icons shows clearly without the need for black stripe. keeping the dock area clean for good readability.
2. its the easiest way to create a lot of wallpapers that have day and night versions. for light and dark modes. you can keep it at the exact position with changings just for the lighting. or a little bit of changes to make it intresting. we have the choice.
it even open the posability to more then 2 modes per wallpaper. (night, sunshine, day, sunrise, evening)
3. its a great way to make a distinguish style to 'elementary wallpapers'. it can be cartoony, it can be foggy landscapes, it can be nature-inspired pastel-colored abstract images. we can keep it realistic like now, and decide about more specific colors or somthing.

its just more control.

I know its a little bit overdo. but I think It might be nice for 3D artist thats like elementary and don't know how to code, and search for ways to contribute.

what do you think?

**Comment by danirabbit** [comment]
I'm in favor of 3D wallpapers if they're done well! We had previously talked a bit about this when first looking into doing the Odin wallpapers. I agree it would be cool if we could developer a pallete and/or set of shaders for Blender or something maybe to get a consistent visual style. Like pie in the sky dreaming here, but that would be really dope
